group.ids

```diff
@@ -1,2 +1,2 @@
 SYSTEM_GROUPS sailfish-system,sailfish-datetime,timed,system,ssu
-USER_GROUPS video,input,audio,users,oneshot,sailfish-alarms,bluetooth,graphics,camera,media,media_rw,mtp,inet
+USER_GROUPS video,input,audio,users,sailfish-alarms,sailfish-phone,sailfish-messages
```

Comment:
If @mkosola thinks that bluetooth... can be now go then I'm ok with this.

Reply:
@rainemak bluetooth group is added here https://git.sailfishos.org/mer-core/bluez5/blob/master/rpm/bluez5.spec#L237

Reply:
that bluez5 line just adds nemo user to bluetooth group.

So we definitely need the bluetooth group here.

Bluez5 probably should install file to /usr/lib/droid/hw-group.d with line "bluetooth".

Reply:
Ok, so bluetooth is needed for now at least. Well, what ensures that bluetooth group is there when user is added to these groups? Adding happens when user session is started or when user is created. That file there sounds like a more robust approach.

Reply:
@jusa  but I'm seeing "bluetooth" line in the /usr/lib/droid/hw-group.d/default.list

Reply:
Well, added bluetooth back. At least for the time being. Are the other groups okay though? They are in the adaptation file but so is audio.

Reply:
Also like, my reasoning for removing these hardware specific groups was that they weren't part of this until this file was added so they shouldn't be necessary. I guess audio and bluetooth are a bit special as they are also needed by native applications.